src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/InitialConsiderationHelper.java

```diff
@@ -0,0 +1,120 @@
+package uk.gov.hmcts.ethos.replacement.docmosis.helpers;
+
+import uk.gov.hmcts.et.common.model.ccd.items.DateListedTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.HearingTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.JurCodesTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.items.RespondentSumTypeItem;
+import uk.gov.hmcts.et.common.model.ccd.types.HearingType;
+import uk.gov.hmcts.ethos.replacement.docmosis.domain.referencedata.JurisdictionCode;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Optional;
+
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.DocumentHelper.getHearingDuration;
+import static uk.gov.hmcts.ethos.replacement.docmosis.helpers.Helper.nullCheck;
+
+public class InitialConsiderationHelper {

```

Comment:
Worth adding a little JavaDoc at the class level like you have an intro in a book!

Reply:
done